app/helpers/users_helper.rb

```diff
@@ -33,14 +33,22 @@ def user_action_buttons(user, additional_actions = [])
         :method => :post,
         :data => { :no_turbolink => true })
     end
+    
+    if user != User.current
+      additional_actions <<  display_link_if_authorized(_("Invalidate JWT"),
```

Comment:
the exact same problem like with `can?` above, the auth object needs to be specified. If this is used on some users listing page, to avoid N+1 issue, an authorized object needs to be used.

Reply:
I am adding `:auth_object` here, is [this](https://github.com/theforeman/foreman/pull/10357/commits/cc036ad9ccfd90905d5dd6d22cd29ad926dc4ae5#diff-0a32d4e9f83e8c59487817bce4ccece334edc27596e9c7c9e8e2888ed4b0fc97R39) what you're talking about?